src/cryptography/hazmat/primitives/ciphers/base.py

```diff
@@ -22,14 +22,6 @@
 from cryptography.hazmat.primitives.ciphers import modes
 
 
-class BlockCipherAlgorithm(metaclass=abc.ABCMeta):
```

Comment:
wut, how did this happen

Reply:
Turns out we had moved it to the private `_cipheralgorithm` module and imported it elsewhere, but left it in `base.py`. This confused me with some isinstance checks for awhile until I realized it wasn't working because they were two separate classes.